### ISSUE TITLE
[Functional-Help-Projects] Help menu gets collapsed after pressing enter key on Projects options inside help menu.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2541]
**User Experience:**
User will not able to access Project screen if Help menu gets collapsed after pressing enter key on Project option inside Help menu.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Microbit section element and select Code control given on it.
3-Navigate to the Help control lying in the header section.
4- Navigate to Projects option inside Help menu and press enter key on it.
5-Verify, If Project dialog appears on the screen after pressing enter key on Projects options inside help menu or not.

**Actual Result**
Help menu gets collapsed after pressing enter key on Projects options inside help menu.

**Expected Result**
Project dialog should appears on the screen after pressing enter key on Projects options inside help menu.

Please Refer attachment for more details
[Functional_Help_projects_menu getting collapsed.zip](https://github.com/Microsoft/pxt/files/1179393/Functional_Help_projects_menu.getting.collapsed.zip)

**Comment by veler** [comment]
We cannot reproduce this issue. Please try to reproduce it again on https://makecode.microbit.org/acc . If you can still reproduce it, please provide us more information on how to reproduce it (the environment, the step by step from when the page is loaded (a video is fine)).

**Comment by abchatra** [comment]
This bug is no repro, Amit please help to get the repro.

**Comment by v-mesala** [comment]
Issue Verified, Now issue doesn't exists. Thus closing the bug.
